src/main/java/com/urbanairship/datacube/dbharnesses/HBaseDbHarness.java

```diff
@@ -130,22 +142,29 @@ public Integer value() {
     }
 
     @Override
-    public Optional<T> get(Address c) throws IOException, InterruptedException  {
-        final byte[] rowKey = ArrayUtils.addAll(uniqueCubeName, c.toKey(idService));
-        
+    public Optional<T> get(Address c) throws IOException, InterruptedException {
```

Comment:
i'm okay with "that change set would be enormous", but what are your thoughts on renaming "c?"